pkgs/development/tools/rust/cargo-llvm-cov/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchCrate, fetchFromGitHub, libllvm, makeBinaryWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-llvm-cov";
+  version = "0.4.14";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-DY5eBSx/PSmKaG7I6scDEbyZQ5hknA/pfl0KjTNqZlo=";
+  };
+
+  postPatch = ''
+    rm tests/test.rs
+  '';
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+  buildInputs = [ libllvm ];
```

Comment:
Should this use the same libllvm as rustc (e.g. `rustPlatform.rust.rustc.llvmPackages.libllvm`)?

Reply:
Not sure if this is different because it's an older version, but I didn't need to add any `buildInputs` at all to get 0.5.0 to build/function.

Reply:
Yeah, looks like libllvm was never required, at least on 0.4.13.

@DieracDelta Do you remember why you added it?

Reply:
I have no recollection of adding this. If it works without libllvm, then let's drop the dependency 👍